impl/src/test/groovy/com/okta/sdk/impl/ds/DefaultDataStoreTest.groovy

```diff
@@ -69,11 +69,13 @@ class DefaultDataStoreTest {
         def response = mock(Response)
         def responseBody = '{"name": "jcoder"}'
         def defaultDataStore = new DefaultDataStore(requestExecutor, "https://api.okta.com/v1", apiKeyResolver)
+        def headers = new HttpHeaders()
```

Comment:
Update the test with `application/ion+json` once it is added per my above comment.